traveler-swift-core/TravelerKit/Models/ProductOffering.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  ProductOffering.swift
+//  TravelerKit
+//
+//  Created by Omar Padierna on 2019-11-23.
+//  Copyright © 2019 Ata Namvari. All rights reserved.
+//
+
+import Foundation
+//TODO: Find a better name for this protocol. Both passes and PartnerOffering conform to it, but it doesn't sound right for passes. 
+ /// An offering for a `Product`
+public protocol ProductOffering {
```

Comment:
Maybe rename this to `PurchaseSummary`.  It seems like our notion of `Pass` is really just a summary of the purchase details.  Not actual tickets (passes) a user can use for admission.

Reply:
No, our notion of pass is an actual pass. When you select adult/child/senior passes in the booking buy flow, you're (behind the scenes) creating pass models and sending them in the body when making an order request.